docs/src/main/sphinx/functions/json.md

```diff
@@ -1146,6 +1146,68 @@ FROM customers
 | 102 | 'missing' |
 | 103 | 'missing' |
 
+
+(json-table)=
+## json_table
+
+The `json_table` table function extracts a table of one or more columns and rows
+of data from a JSON value.
+
+```text
+JSON_TABLE(
+  json_input [ FORMAT JSON [ ENCODING { UTF8 | UTF16 | UTF32 } ] ],
+)
+```
+
+
```

Comment:
One example which i put together:

```
trino> SELECT
    ->   *
    -> FROM
    ->   json_table (
    ->     '[{"id":1,"name":"Africa","wikiDataId":"Q15"},{"id":2,"name":"Americas","wikiDataId":"Q828"},{"id":3,"name":"Asia","wikiDataId":"Q48"},{"id":4,"name":"Europe","wikiDataId":"Q51"}]',
    ->     'lax $' COLUMNS (
    ->         NESTED PATH 'lax $[*]' AS nested_path COLUMNS (
    ->             id integer path 'lax $[*].id',
    ->             name varchar path 'lax $[*].name',
    ->             wiki_data_id varchar path 'lax $[*]."wikiDataId"')));
 ID |   NAME   | WIKI_DATA_ID 
----+----------+--------------
  1 | Africa   | Q15          
  2 | Americas | Q828         
  3 | Asia     | Q48          
  4 | Europe   | Q51          
(4 rows)
```

Reply:
You don't need the `[*]` in `$[*].id`, `$[*].name` and '$[*].wikiDataId`, since the elements of the top-level array are not arrays. It happens to work because you're using `lax`, which does some conversions automatically and is more lenient. This would be the proper form:

```
SELECT
  *
FROM
  json_table (
	'[{"id":1,"name":"Africa","wikiDataId":"Q15"},{"id":2,"name":"Americas","wikiDataId":"Q828"},{"id":3,"name":"Asia","wikiDataId":"Q48"},{"id":4,"name":"Europe","wikiDataId":"Q51"}]',
	'strict $' COLUMNS (
		NESTED PATH 'strict $[*]' AS nested_path COLUMNS (
			id integer path 'strict $.id',
			name varchar path 'strict $.name',
			wiki_data_id varchar path 'strict $."wikiDataId"')));
```

Reply:
Thanks @findinpath and @martint .. I barely started but I will look at the various test cases and such to create some examples as well as details the different sections of the table function.